build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.1.2'
+        classpath 'com.android.tools.build:gradle:2.2.2'
```

Comment:
While this is a valid change, it violates separation of concerns: it's not necessary for implementing around filters and it's not related to the original problem. As such, it must not be part of this PR, but rather a separate one.